pilot/utils/exit.py

```diff
@@ -84,6 +101,16 @@ def exit_gpt_pilot(project, ask_feedback=True):
 
     ask_to_store_prompt(project, path_id)
 
-    ask_user_feedback(project, path_id, ask_feedback)
+    # Ask user for contact email. If they choose not to provide it, ask for direct
+    # feedback. Both are optional.
+    if not ask_user_email(project, path_id, ask_feedback):
+        ask_user_feedback(project, path_id, ask_feedback)
```

Comment:
I would avoid asking him twice. I am ok with turning off feedback for now since we gathered a lot of it